guava/hw10-workHibernate/src/main/java/ru/otus/core/model/User.java

```diff
@@ -0,0 +1,81 @@
+package ru.otus.core.model;
+
+
+
+import javax.persistence.*;
+
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "name")
+    private String name;
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JoinColumn(name="id")
+    private AddressDataSet street;
+    @OneToMany(fetch = FetchType.LAZY)
+    @JoinColumn(name="id")
```

Comment:
```suggestion
    @JoinColumn(name="user_id")
```
Посмотрите в логах на CREATE TABLE. Соответствует ли создаваемая структура таблиц тому, что вы ожидаете получить?